val1 = 10
val2 = 5
try:
    a = val1/val2
except Exception as e:
    print(e)
else:
    print('no errors!')
